oak_marketplace_emotion_recognition/README.md

```diff
@@ -0,0 +1,51 @@
+This contains the code for **Emotion Recognition using OpenCV AI KIT and Modelplace**. For more information - visit [OpenCV.AI](https://www.opencv.ai/).
+
+## Installation:
+
+1. Clone [repository](https://github.com/opencv-ai/oak-model-samples) with OAK model samples by the following comand:
+
+    ```git clone https://github.com/opencv-ai/oak-model-samples.git```
+
+2. Set up Emotion Recognition package by following commands:
+
+   2.1 `cd emotion_recognition_retail`
+
+   2.2 `python3 setup.py bdist_wheel && rm -R build/ *.egg-info`
+
+   2.3 `pip3 install dist/*.whl -f https://artifacts.luxonis.com/artifactory/luxonis-python-snapshot-local/depthai/ && rm -R dist/`
+
+## Usage
+
```

Comment:
The usage should be simpler like 

To make the code working as described in a blog post, run it with the following command:

```bash
python main.py -model <PATH_TO_EMOTION_RECOGNITION_MODEL>
```